hippo4j-common/src/main/java/cn/hippo4j/common/api/IExecutorProperties.java

```diff
@@ -17,6 +17,8 @@
 
 package cn.hippo4j.common.api;
 
+import cn.hippo4j.common.config.ExecutorNotifyProperties;
```

Comment:
Because `IExecutorProperties.class` was moved to the api subpackage under the common module, and the `ExecutorNotifyProperties.class` was moved to the config subpackage under the common module. 
Originally, these two classes were located under the spring-boot-starter package.
Actually, these classes have nothing to do with spring.